sdk/eventhub/azure-eventhub-checkpointstoretable/tests/test_storage_table_partition_manager.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+from azure.eventhub.extensions.checkpointstoretable import TableCheckpointStore
+
+
+def test_constructor():
+    client = TableCheckpointStore()
+    assert client is not None
```

Comment:
nit - add newline to end of file:
```suggestion
    assert client is not None
    
```